takoyaki/src/main/java/frc/robot/subsystems/Shooter/ClosedLoopSingleFalconShooter.java

```diff
@@ -35,7 +35,7 @@ public ClosedLoopSingleFalconShooter() {
 
     left.setInverted(TalonFXInvertType.CounterClockwise);
     right.setInverted(TalonFXInvertType.Clockwise);
-    kicker.setInverted(Constants.kShooter.KICKER_INVERSION);
+    kicker.setInverted(Constants.kShooter.kKicker.KICKER_INVERSION);
```

Comment:
why did you make the kicker a new subclass? its part of the shooter subsystem but also this could just be better so i am down to leave it

Reply:
I'm not sure but there were a lot of kicker constants in kShooter so I grouped them together